src/main/java/com/onarandombox/MultiverseNetherPortals/listeners/MVNPPlayerListener.java

```diff
@@ -35,25 +36,19 @@ public MVNPPlayerListener(MultiverseNetherPortals plugin) {
         this.linkChecker = new MVLinkChecker(this.plugin);
     }
 
-    @EventHandler(priority = EventPriority.NORMAL)
+    @EventHandler(priority = EventPriority.NORMAL, ignoreCancelled = true)
     public void onPlayerPortal(PlayerPortalEvent event) {
-        if (event.isCancelled()) {
-            this.plugin.log(Level.FINEST, "PlayerPortalEvent was cancelled! NOT teleporting!");
-            return;
-        }
```

Comment:
My only note about this is that we lose the logging. However, I'm not sure if this particular log line is very helpful.